airbyte-integrations/connectors/source-bigquery/acceptance-test-config.yml

```diff
@@ -20,9 +21,9 @@ acceptance_tests:
         configured_catalog_path: "integration_tests/configured_catalog.json"
         empty_streams: []
```

Comment:
If there are no `empty_streams` - we shouldn't specify this.

Reply:
done here https://github.com/artefactory/airbyte/pull/28/commits/70f624cde28a92c777bd6db4515f36b0fbc8c908